README.md

```diff
@@ -160,8 +160,7 @@ kubectl exec -it mongo-0 -- mongo <samples/mongo/replica.mongo
 # Add razeedash secret for mongo_url
 kubectl apply -f samples/mongo/secret.yaml
 # Get latest release of razeedash-api and deploy
-VER=$(git describe --tags `git rev-list --tags --max-count=1`) \
-  curl -L "https://github.com/razee-io/razeedash-api/releases/download/${VER}/resource.yaml"
+curl -o /tmp/release.yaml -L "https://github.com/razee-io/razeedash-api/releases/latest/download/resource.yaml"
```

Comment:
Will kubectl follow the redirect? 
ie can we just do 
`kubectl apply -f  "https://github.com/razee-io/razeedash-api/releases/latest/download/resource.yaml"`

Reply:
updated